src/utils/autoScroll.js

```diff
@@ -0,0 +1,10 @@
+const TOP = { x: 0, y: 0 };
+
+function autoScroll(options) {
+  window.scrollTo(options.x, options.y);
```

Comment:
Oh, there are two other places in our code that does this:

window.scroll({
        top: tabbedContainer.current.offsetTop,
        left: 0,
        behavior: "smooth",
      });

Can we use window.scroll here (it uses smoothscroll-polyfill) and update the other uses to use your file?

Reply:
Will do!